### ISSUE TITLE
For all orgs with status "static pooling" change subit to epaad logic

### ISSUE THREAD

**Issue by viandrea** [openLAMA/Welcome#100]
Valid only for Orgy Type: Camps
User: University Admin

Add switch for org type camp:  Epaad Done

Fields to be added in the email which is send to a group mailbox mentioned below
- Pooling Name 
- Country / 
- Canton / 
- ZIP / 
- City / 
- Street / 
- Email / 
- Number of Pools

Adress for Email recipient: breitestesten@bl.ch

**Comment by TsvyatkoIvanov** [comment]
I was thinking about it.
The "Send to epaad" button will open a modal that will say something like "sends an email with this information, when data has be saved to epaad, please press the switch". This means that they will be able to click the "Send to epaad" button as many times as they want until they confirm the data was saved to epaad. 
In this case using a switch to confirm is not a good idea, because the switch is designed to be turned on and off multiple times. But in our case if they click it once they should not be able to return it back. I have to add a modal to confirm this action because it's a one time action.
I think using a button like the "send to epaad" is better in this case.
If we leave it as a switch, they can return it back to "off" position which means that the data was not saved in epaad and the "send to epaad" button should be active again.
What do you think. @viandrea ?

**Comment by viandrea** [comment]
I suggest we just to the "send email" part, and wait for feedback from Anouk.
Maybe they wont need to indicate the epaad status.

**Comment by tockata** [comment]
@viandrea would you please translate these /for the email template/, and may be we need to add the organization name too:

Pooling Name
Country /
Canton /
ZIP /
City /
Street /
Email /
Number of Pools

And also please tell me what the subject should be. Thanks!

**Comment by tockata** [comment]
@viandrea which organization email we should add in the email body? Selected by university admin or just add all org.contacts emails?

I am doing it for now by adding all org. emails.

**Comment by viandrea** [comment]
Makes sense, let's add all. The admin can the decide which one to use, in case there are several...

**Comment by tockata** [comment]
@viandrea here is is how the email looks like now. I added the organization name and "Active Since" - which is the creation date of the organization /I saw it is send to epaad when pushing to epaad and may be they need it too/. Let me know if I have to change something:

![image](https://user-images.githubusercontent.com/7578157/124269674-df440200-db43-11eb-938a-a54b4f1cee4e.png)

**Comment by viandrea** [comment]
Great!
